[2022-07-06 21:44:29,648] {processor.py:153} INFO - Started process (PID=158969) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:44:29,648] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:44:29,649] {logging_mixin.py:115} INFO - [2022-07-06 21:44:29,649] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:44:29,665] {logging_mixin.py:115} INFO - [2022-07-06 21:44:29,665] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:44:29,665] {logging_mixin.py:115} INFO - [2022-07-06 21:44:29,665] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:44:29,665] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:44:29,674] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2022-07-06 21:45:03,623] {processor.py:153} INFO - Started process (PID=159052) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:03,624] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:45:03,624] {logging_mixin.py:115} INFO - [2022-07-06 21:45:03,624] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:03,637] {logging_mixin.py:115} INFO - [2022-07-06 21:45:03,637] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:45:03,637] {logging_mixin.py:115} INFO - [2022-07-06 21:45:03,637] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:45:03,637] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:03,647] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:45:38,211] {processor.py:153} INFO - Started process (PID=159137) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:38,212] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:45:38,213] {logging_mixin.py:115} INFO - [2022-07-06 21:45:38,213] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:38,226] {logging_mixin.py:115} INFO - [2022-07-06 21:45:38,225] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:45:38,226] {logging_mixin.py:115} INFO - [2022-07-06 21:45:38,226] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:45:38,226] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:45:38,234] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:46:12,723] {processor.py:153} INFO - Started process (PID=159239) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:12,724] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:46:12,725] {logging_mixin.py:115} INFO - [2022-07-06 21:46:12,724] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:12,738] {logging_mixin.py:115} INFO - [2022-07-06 21:46:12,738] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:46:12,738] {logging_mixin.py:115} INFO - [2022-07-06 21:46:12,738] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:46:12,738] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:12,747] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:46:47,982] {processor.py:153} INFO - Started process (PID=159312) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:47,983] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:46:47,983] {logging_mixin.py:115} INFO - [2022-07-06 21:46:47,983] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:47,996] {logging_mixin.py:115} INFO - [2022-07-06 21:46:47,996] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:46:47,997] {logging_mixin.py:115} INFO - [2022-07-06 21:46:47,997] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:46:47,997] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:46:48,006] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:47:22,999] {processor.py:153} INFO - Started process (PID=159401) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:23,000] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:47:23,000] {logging_mixin.py:115} INFO - [2022-07-06 21:47:23,000] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:23,015] {logging_mixin.py:115} INFO - [2022-07-06 21:47:23,014] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:47:23,015] {logging_mixin.py:115} INFO - [2022-07-06 21:47:23,015] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:47:23,015] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:23,024] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2022-07-06 21:47:57,868] {processor.py:153} INFO - Started process (PID=159488) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:57,869] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:47:57,870] {logging_mixin.py:115} INFO - [2022-07-06 21:47:57,870] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:57,885] {logging_mixin.py:115} INFO - [2022-07-06 21:47:57,885] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:47:57,885] {logging_mixin.py:115} INFO - [2022-07-06 21:47:57,885] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:47:57,898] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:47:57,915] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2022-07-06 21:48:31,173] {processor.py:153} INFO - Started process (PID=159571) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:48:31,173] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:48:31,174] {logging_mixin.py:115} INFO - [2022-07-06 21:48:31,174] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:48:31,187] {logging_mixin.py:115} INFO - [2022-07-06 21:48:31,186] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:48:31,187] {logging_mixin.py:115} INFO - [2022-07-06 21:48:31,187] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:48:31,187] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:48:31,196] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2022-07-06 21:49:05,300] {processor.py:153} INFO - Started process (PID=159664) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:05,301] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:49:05,301] {logging_mixin.py:115} INFO - [2022-07-06 21:49:05,301] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:05,314] {logging_mixin.py:115} INFO - [2022-07-06 21:49:05,314] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:49:05,314] {logging_mixin.py:115} INFO - [2022-07-06 21:49:05,314] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:49:05,315] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:05,323] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2022-07-06 21:49:38,362] {processor.py:153} INFO - Started process (PID=159768) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:38,363] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:49:38,363] {logging_mixin.py:115} INFO - [2022-07-06 21:49:38,363] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:38,377] {logging_mixin.py:115} INFO - [2022-07-06 21:49:38,376] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:49:38,377] {logging_mixin.py:115} INFO - [2022-07-06 21:49:38,377] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:49:38,377] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:49:38,385] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:50:14,458] {processor.py:153} INFO - Started process (PID=159849) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:14,459] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:50:14,459] {logging_mixin.py:115} INFO - [2022-07-06 21:50:14,459] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:14,473] {logging_mixin.py:115} INFO - [2022-07-06 21:50:14,472] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:50:14,473] {logging_mixin.py:115} INFO - [2022-07-06 21:50:14,473] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:50:14,473] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:14,481] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2022-07-06 21:50:47,704] {processor.py:153} INFO - Started process (PID=159925) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:47,705] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:50:47,705] {logging_mixin.py:115} INFO - [2022-07-06 21:50:47,705] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:47,719] {logging_mixin.py:115} INFO - [2022-07-06 21:50:47,719] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:50:47,719] {logging_mixin.py:115} INFO - [2022-07-06 21:50:47,719] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:50:47,719] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:50:47,728] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:51:22,807] {processor.py:153} INFO - Started process (PID=160010) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:22,808] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:51:22,808] {logging_mixin.py:115} INFO - [2022-07-06 21:51:22,808] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:22,822] {logging_mixin.py:115} INFO - [2022-07-06 21:51:22,822] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:51:22,822] {logging_mixin.py:115} INFO - [2022-07-06 21:51:22,822] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:51:22,822] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:22,831] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2022-07-06 21:51:55,504] {processor.py:153} INFO - Started process (PID=160093) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:55,505] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:51:55,505] {logging_mixin.py:115} INFO - [2022-07-06 21:51:55,505] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:55,520] {logging_mixin.py:115} INFO - [2022-07-06 21:51:55,519] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:51:55,520] {logging_mixin.py:115} INFO - [2022-07-06 21:51:55,520] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:51:55,520] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:51:55,530] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2022-07-06 21:52:27,547] {processor.py:153} INFO - Started process (PID=160174) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:52:27,547] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:52:27,547] {logging_mixin.py:115} INFO - [2022-07-06 21:52:27,547] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:52:27,561] {logging_mixin.py:115} INFO - [2022-07-06 21:52:27,560] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:52:27,561] {logging_mixin.py:115} INFO - [2022-07-06 21:52:27,561] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:52:27,561] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:52:27,570] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2022-07-06 21:53:00,947] {processor.py:153} INFO - Started process (PID=160259) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:00,949] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:53:00,949] {logging_mixin.py:115} INFO - [2022-07-06 21:53:00,949] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:00,963] {logging_mixin.py:115} INFO - [2022-07-06 21:53:00,962] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:53:00,963] {logging_mixin.py:115} INFO - [2022-07-06 21:53:00,963] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:53:00,963] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:00,972] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2022-07-06 21:53:34,528] {processor.py:153} INFO - Started process (PID=160342) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:34,529] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:53:34,529] {logging_mixin.py:115} INFO - [2022-07-06 21:53:34,529] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:34,549] {logging_mixin.py:115} INFO - [2022-07-06 21:53:34,549] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:53:34,549] {logging_mixin.py:115} INFO - [2022-07-06 21:53:34,549] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:53:34,549] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:53:34,558] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2022-07-06 21:54:09,715] {processor.py:153} INFO - Started process (PID=160430) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:09,716] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:54:09,717] {logging_mixin.py:115} INFO - [2022-07-06 21:54:09,717] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:09,733] {logging_mixin.py:115} INFO - [2022-07-06 21:54:09,733] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:54:09,733] {logging_mixin.py:115} INFO - [2022-07-06 21:54:09,733] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:54:09,733] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:09,744] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2022-07-06 21:54:42,671] {processor.py:153} INFO - Started process (PID=160546) to work on /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:42,671] {processor.py:641} INFO - Processing file /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-07-06 21:54:42,671] {logging_mixin.py:115} INFO - [2022-07-06 21:54:42,671] {dagbag.py:507} INFO - Filling up the DagBag from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:42,685] {logging_mixin.py:115} INFO - [2022-07-06 21:54:42,685] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-07-06 21:54:42,685] {logging_mixin.py:115} INFO - [2022-07-06 21:54:42,685] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-07-06 21:54:42,686] {processor.py:653} WARNING - No viable dags retrieved from /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-07-06 21:54:42,694] {processor.py:161} INFO - Processing /home/nicholas/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
                                                                                                                                                                                                                                                                                                                            